src/main/java/de/uniba/dsg/serverless/azure/AzureLogHandler.java

```diff
@@ -1,6 +1,164 @@
 package de.uniba.dsg.serverless.azure;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.InvalidKeyException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.file.CloudFile;
+import com.microsoft.azure.storage.file.CloudFileClient;
+import com.microsoft.azure.storage.file.CloudFileDirectory;
+import com.microsoft.azure.storage.file.CloudFileShare;
+import com.microsoft.azure.storage.file.ListFileItem;
+
+import de.uniba.dsg.serverless.model.PerformanceData;
+import de.uniba.dsg.serverless.model.SeMoDeException;
+
 public class AzureLogHandler {
+
+	private static final String FUNCTION_COMPLETED = "Function completed";
+
+	private static final String FUNCTION_STARTED = "Function started";
+
+	private static final Logger logger = Logger.getLogger(AzureLogHandler.class.getName());
+
+	public final String storageConnectionString;
```

Comment:
Make fields private, especially the connection string is a potential security issue. Fields are not used in other classes, therefore no access is needed.